pkg/operator/ceph/object/rgw.go

```diff
@@ -200,10 +200,9 @@ func (c *clusterConfig) startRGWPods() error {
 				logger.Warningf("failed to delete rgw secret %q. %v", secretToRemove, err)
 			}
 
-			// Delete the auth key
-			err = cephclient.AuthDelete(c.context, c.store.Namespace, generateCephXUser(depNameToRemove))
+			err := c.deleteRgwCephObjects(depNameToRemove)
 			if err != nil {
-				logger.Infof("failed to delete rgw key %q. %v", depNameToRemove, err)
+				logger.Errorf("%v", err)
```

Comment:
If it's not going to fail the reconcile, a warning seems more consistent
```suggestion
				logger.Warningf("%v", err)
```